pkg/envtest/webhook.go

```diff
@@ -64,6 +65,11 @@ type WebhookInstallOptions struct {
 	// it will be automatically populated by the local temp dir
 	LocalServingCertDir string
 
+	// CustomTLSConfig is a *tls.Config object that will be used in place of
+	// the one normally created by reading certs in the CertDir.  If both
+	// CertDir and this are specified CertDir will be ignored
+	CustomTLSConfig *tls.Config
```

Comment:
Can we change this here and in the manager to return an error when both `CertDir` and `CustomTLSConfig` are specified to avoid hard to debug issues?